testing_script/variables.py

```diff
@@ -1,22 +1,23 @@
 acceptCookie = "hs-eu-confirmation-button"
 careersButton = "Careers"
 locationDrobBox = "//span[@class='label' and text()='Location']"
-selectingVilnius = "//*[@id='post-108']/div/div[1]/div[2]/div[1]/div/div[1]/div[3]/div/ul/li[10]"
-categoryDropBox = "//*[@id='post-108']/div/div[1]/div[2]/div[1]/div/div[2]/div[2]/b"
-selectingSales = '//*[@id="post-108"]/div/div[1]/div[2]/div[1]/div/div[2]/div[3]/div/ul/li[6]'
-searchForJobs = "//button[contains(text(), 'Search')]"
-technologyAndEngineeringCheckBox = "//*[@id='search-filter']/div[4]/div/div[4]"
-seniorTestAutomationEngineerPosition = 'Senior'
-frameName = 'grnhse_iframe'
-inputName = 'first_name'
-inputLastName = 'last_name'
-inputMail = 'email'
-inputPhone = 'phone'
```

Comment:
Jei tai yra string values, gali perkelti i kita atskira faila (kaip variables, konstantas). 
Jeigu tai yra xpath, tai pakeisk pavadinima (pvz. inputNameElementId)
del to gereja skaitomumas